src/arrayReverse.test.js

```diff
@@ -8,13 +8,42 @@ describe(`Function 'arrayReverse':`, () => {
   });
 
   it(`should return an array`, () => {
+    expect(arrayReverse([])).toEqual([]);
+  });
 
+  it(`should return an empty string`
+    + `, if original array consists of an empty string`, () => {
```

Comment:
The test description mentions returning an empty string, but the test case checks for an array of empty strings. Consider updating the description to accurately reflect the test case.

Reply:
fixed